cypress/integration/cbl/progress/overrides.js

```diff
@@ -0,0 +1,370 @@
+
+    it('Override a skill with an M rating and one white box.', () => {
+        loadDashboard('SS', 'group:class_of_2020');
+
+        let competency = 32; // SS.4
+        let skill = 169; // SS.4.1
+        let student = 4; // student
+
+        overrideSkill(competency, skill, student);
+
+        // check portfolio value for graduation at cell at SS.4 - student
+        cy.get('.cbl-grid-main .cbl-grid-progress-row[data-competency="'+competency+'"]')
+            .should('have.length', 1)
+            .find('.cbl-grid-progress-cell[data-student="'+student+'"]')
+                .should('have.length', 1)
+                .should('have.class', 'cbl-level-10')
+                .find('.cbl-grid-progress-percent')
+                    .should('contain.text', '0%')
+    });
+
+    it('Override for all skills where one set of ERs has all Ms.', () => {
+        loadDashboard('SS', 'group:class_of_2020');
+
+        let competency = 32; // SS.4
+        let skill = 170; // SS.4.2
+        let student = 6; // student2
+
+        overrideSkill(competency, skill, student);
+
+        // check portfolio value for graduation at cell at SS.4 - student
+        cy.get('.cbl-grid-main .cbl-grid-progress-row[data-competency="'+competency+'"]')
+            .should('have.length', 1)
+            .find('.cbl-grid-progress-cell[data-student="'+student+'"]')
+                .should('have.length', 1)
+                .should('have.class', 'cbl-level-10')
+                .find('.cbl-grid-progress-percent')
+                    .should('contain.text', '0%')
+    });
+
+    it('Submit a rating for a skill with an override.', () => {
+        loadDashboard('ELA', 'group:class_of_2020');
+
+        let competency = 6; // ELA.6
+        let skill = 28; // ELA.6.1
+        let student = 6; // student2
+
+        // expand competency
+        cy.get('.cbl-grid-progress-row[data-competency="'+competency+'"] .cbl-grid-competency-name')
+            .should('have.length', 1)
+            .click();
+
+        // click cell for given skill/student
+        cy.get('.cbl-grid-main .cbl-grid-skills-row[data-competency="'+competency+'"]')
+            .should('have.length', 1)
+            .should('have.class', 'is-expanded')
+            .find('.cbl-grid-skill-row[data-skill="'+skill+'"] .cbl-grid-demos-cell[data-student="'+student+'"]')
+                .click();
+
+        // wait for window to transition open
+        cy.extGet('slate-window title[text="Skill History"]')
+            .should('not.have.class', 'x-hidden-clip')
+            .within(($window) => {
+                cy.extGet('button[text="Submit Evidence"]')
+                    .should('exist')
+                    .click();
+            });
+
+        // wait for window to transition open
+        cy.extGet('slate-window title[text="Submit Evidence"]')
+            .should('not.have.class', 'x-hidden-clip')
+            .within(($window) => {
+                // TODO: select using combo rather than typing
```

Comment:
@BillClinton I'd drop this TODO and the next one—using the combo would be more complex and we don't need to be testing the picker